BD-2409: Quick Push EA (#6109)
braze-inc/braze-docs: _docs/_hidden/private_betas/quick_push.md

---
nav_title: Quick Push Campaigns
article_title: Quick Push Campaigns
permalink: "/quick_push/"
hidden: true
---

# Quick push campaigns

When creating a push campaign in Braze, you can select multiple platforms and devices to craft one message for all platforms in a single editing experience.

{% alert important %}
This functionality is currently in early access and is only available for campaigns.
{% endalert %}

## Use cases

This editing experience is best for the following use cases:

- Mobile push campaigns that need to be sent to multiple device types (such as both iOS and Android).
- Time-sensitive push notifications that need to target multiple platforms quickly and accurately, where content is the same across platforms (such as breaking news or live game updates).

## What's different

> Overall, the process for creating your push message is the same. However, if you select multiple platforms or devices, your composer will be a bit different from normal.

Reviewer: ```suggestion
Overall, the process for creating your push message is the same. However, if you select multiple platforms or devices, your composer will look slightly different from usual.
```
?

![Options to select multiple platforms for a push campaign, such as Mobile, Web, and Kindle, and multiple devices, such as iOS and Android.][1]

{% alert note %}

> After you click **Next**, you will not be able to change your selected platforms or devices.

Reviewer: ```suggestion
After clicking **Next**, you cannot change your selected platforms or devices.
```
or "will be unable to"
